samples/unity-of-bugs/Assets/Scripts/SceneLoaderBehaviour.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Assets.Scripts

```

Comment:
We have two scripts in samples: One lives inside a namespace, the other doesn't.

Reply:
+